.travis.yml

```diff
@@ -0,0 +1,31 @@
+sudo: false    #container 
+
+language: python
+python:
+  - "2.7"
+  - "3.6"
+  # - "3.7"  fails build 'Unable to download 3.7 archive. The archive may not exist.''
+
+notifications:
+  email: never
+
+install:
+  - pip install -r requirements.txt
+  - pip install codecov
+  # - pip install pytest
+  - pip install pytest pytest-cov
+
+#virtualenv:
+#  system_site_packages: true
+
+env:
+  - CODECOV_TOKEN="3b343171-4511-44bd-a4d6-dd7d64aef700"  #get token from https://codecov.io/ --> add new repository
+
+script:
+  - py.test --cov-report xml 
+  - coverage run fatf/tests/test_script.py
```

Comment:
Will this only run coverage tests on the tests in test_script.py and not the ones in the subdirectories like *fatf/metrics/tests*. Also is this line necessary as the previous like will produce a coverage report in the local directory.

Reply:
I'll try change it to run all files in directory, perhaps using wildcard? something like `coverage run fatf/metrics/tests/*.py`

the coverage line is to set up with codecov.io to see the coverage report in a nice format there. I didnt know pytest did coverage reports, but i imagine that would be hard to access if it saves report in local directory when doing the multiple python CI builds? So guess this is why coverage run ... makes this a bit easier?